wsclient/src/main/java/org/hpccsystems/ws/client/HPCCFileSprayClient.java

```diff
@@ -1004,9 +1004,8 @@ public ProgressResponseWrapper sprayVariable(DelimitedDataOptions options, DropZ
         if (targetDropZone == null) throw new Exception("TargetDropZone object not available!");
 
         SprayVariable request = new SprayVariable();
-
         request.setSourceIP(targetDropZone.getNetAddress());
-        request.setSourcePath(targetDropZone.getPath() + "/" + sourceFileName);
+        request.setSourcePath(Utils.ensureTrailingPathSlash(targetDropZone.getPath()) + sourceFileName);
```

Comment:
Do these calls to ensureTrailingPathSlash also need the call to trim()?

Reply:
it's been the user's responsibility to do so, added a java11 instruction to strip all trailing white space chars

Reply:
as discussed offline, we will continue to target java8 for now.
I added a custom trimRight function instead.